unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/EmotesWheelHUD/EmotesWheelView.cs

```diff
@@ -69,7 +65,7 @@ public void SetVisiblity(bool visible)
             {
                 foreach (GameObject go in gameObjectsToHideWhenCustomizeFFIsDeactivated)
```

Comment:
We dont need this flow any longer, probably we can set the GOs in the prefab active.

Reply:
Done!